### DISCUSSION TITLE
Photoprism, share photo

### DISCUSSION THREAD

**Discussion by OsipovOV** [tteck/Proxmox#1478] [Help Needed] [answered]
When I try to share a photo, photoprism enters the default URL - http://photoprism.me:2342 - and won't let me change it. Where can I change it to my domain?

**Comment by tteck** [comment, marked as answer]
https://docs.photoprism.app/getting-started/config-options/#site-information

Set `PHOTOPRISM_SITE_URL` in `/var/lib/photoprism/.env`

**Reply by OsipovOV** [reply]
Thanks!
